tools/gbench/report.py

```diff
@@ -232,7 +240,7 @@ def find_test(name):
             fmt_str = "{}{:<{}s}{endc}{}{:+16.4f}{endc}{}{:+16.4f}{endc}{:14.0f}{:14.0f}{endc}{:14.0f}{:14.0f}"
 
             def get_color(res):
-                if res > 0.05:
+                if res > UTEST_DEFAULT_ALPHA:
```

Comment:
Magic numbers are fun, aren't they? :)
Although this is also `0.05`, this is *not* `UTEST_DEFAULT_ALPHA`,
these are used to color the change of the timers, note the `elif`.

Reply:
Ok, I misunderstood this one then.